pkg/apis/scheduling/types.go

```diff
@@ -195,6 +195,10 @@ type PodGroupSpec struct {
 	// NetworkTopology defines the NetworkTopology config, this field works in conjunction with network topology feature and hyperNode CRD.
 	// +optional
 	NetworkTopology *NetworkTopologySpec
+
+	// TaskNetworkTopology defines the NetworkTopology config for each task, this field works in conjunction with network topology feature and hyperNode CRD.
+	// +optional
+	TaskNetworkTopology map[string]*NetworkTopologySpec
```

Comment:
use label selector

Reply:
https://github.com/volcano-sh/apis/blob/4867e3a42c642a64186d2b03135a15e3f722a990/pkg/apis/topology/v1alpha1/hypernode_types.go#L120

Reply:
network topology > priority > uid 
Add docs

Reply:
Add annotations for configurations.